src/schemas/schemas.ts

```diff
@@ -0,0 +1,24 @@
+
+import S from 'fluent-json-schema';
+
+export const uuid = S.string().pattern(
+  '^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$',
+);
+
+export const id = S.object().prop('id', uuid).required(['id']);
+
+// schema for getting item analytics with view and requestedSampleSize query parameters
+const getOne = {
+  params: id,
+  querystring: {
+    requestedSampleSize: { type: 'string', required: ['requestedSampleSize'] },
+    view: { type: 'string', required: ['view'] }
```

Comment:
Is it possible to limit the value to a given array? You could limit to VIEWS here.

Reply:
I have updated the schema:
- The `view` parameter is limited by the View values.
- The `requestedSampleSize` parameter must be a number and limited between min and max.